android/CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ Have another idea for how you can contribute? Let us know!
 For each our projects, we communicate with the project's:
 - Issues tracker
 - Mailing list
-- [IRC](https://wiki.mozilla.org/IRC) channel. *We're available Monday-Friday
+- [Matrix Fenix](https://chat.mozilla.org/#/room/#fenix:mozilla.org) channel. *We're available Monday-Friday
```

Comment:
Technically this doc is independent of any specific project or application but it's a much better user experience to get linked directly into a chat room than to read a manual so this seems okay to me until we have better options.